packages/react-dom/package.json

```diff
@@ -1,23 +1,26 @@
 {
   "name": "test-react-dom",
   "private": "true",
-  "version": "16.14.0",
+  "version": "0.0.0",
```

Comment:
We don't need to try and keep these in sync. These will never change as the publish script just reads the version value from the field in dependencies and then uses the Eik cli programmatic api library to publish.